pkg/crds/crds.go

```diff
@@ -61,7 +61,7 @@ const (
 	CephResourceCephClusters = "cephclusters"
 	toolBoxDeployment        = "rook-ceph-tools"
 	timeOutCheckResources    = 5 * time.Second
-	maxExecutionTime         = 10 * time.Minute
+	maxExecutionTime         = 15 * time.Minute
```

Comment:
i didnt checked that code can you point me to that, I believe we still can use kubectl wait there

Reply:
so let me ask this, why you are suggesting to use `kubectl wait`? to wait for the resources to be deleted. right? but the above code that you mentioned is the time limit to wait for resources to be deleted after which context will be cancelled.

Reply:
I saw the code now it makes sense, 
But just for to understand why we need a check with an infinite loop for deleting the resources, I have never seen this approach in rook operator or we do similar like there too?

Reply:
you see this line https://github.com/rook/kubectl-rook-ceph/blob/e8e209a623c03b774d0578647552abb1497b4afc/pkg/crds/crds.go#L181-L184

in the context we are setting the timeout after that it will come out so it is not infinite loop techinally